src/Clock.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+
+type Props = {
+  name: string;
+};
+
+type State = {
+  time: string;
+};
+
+function getData(): string {
+  return new Date().toUTCString().slice(-12, -4);
+}
+
+export class Clock extends React.Component<Props, State> {
+  timerId = 0;
+
+  state: Readonly<State> = {
+    time: getData(),
+  };
+
+  handleTimerChange = () => {
+    this.timerId = window.setInterval(() => {
+      const newTime = getData();
```

Comment:
You can change the variable name to time and use it to set in setState
```suggestion
      const time= getData();
```